UI/FleetWnd.cpp

```diff
@@ -978,7 +977,18 @@ namespace {
             auto icon = GG::Wnd::Create<StatisticIcon>(entry.second, 0, 0, false, StatIconSize().x, StatIconSize().y);
             m_stat_icons.push_back({entry.first, icon});
             AttachChild(icon);
-            icon->InstallEventFilter(shared_from_this());
+            icon->RightClickedSignal.connect([entry](const GG::Pt& pt) {
+                std::string meter_string = boost::lexical_cast<std::string>(entry.first);
+
+                auto zoom_article_action = [&meter_string]() { ClientUI::GetClientUI()->ZoomToMeterTypeArticle(meter_string);};
```

Comment:
This results in double popup menus from both the icon and the containing fleet panel. First the icon Pedia Lookup X command appears, then after that's closed, a list of fleet commands appears. The latter shouldn't appear if the click was on the icon.

Reply:
Confirmed.

Reply:
Fixed via 4574e50ccff2134536e036a3478bec08c7cfd80e